### PR TITLE
Order participants list with php

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi.
Here I sort the list of therapy group participants with php instead datatable that not work correctly.
Now in the head of the list shows the active participants and in the end inactive.  

Thanks
Amiel

### CODE REVIEW

File: interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php [hunk 1]
```diff
@@ -192,7 +192,7 @@
                 { "width": "35%", "targets": 5 }
             ],
             "pageLength":6,
-            "order": [[ 0, "asc" ],[ 2, "desc" ]],
+            "order": false,
```
Comment (bradymiller) [inline interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php]: lets put a comment here that this wasn't working, so did the sorting in the php code.

**Comment by bradymiller** [comment]
Is this something that we could fix by using a more recent version of datatable (ie. is this a bug in datatable that has been fixed)?

**Comment by matrix-amiel** [comment]
I tested it with last version of datatable and it still doesn't work..

**Comment by bradymiller** [comment]
thanks for trying most recent version of datatable. will bring your fix into the codebase. thanks, -brady

**Pull request merged**